### PR TITLE
GTW-5220: Update Documentation

### PR DESCRIPTION
**Pull request by duncan-scuffell-cko**
# Description of changes in PR

Add required fields for Level2 and Level3 data

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [x] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Comment by duncan-scuffell-cko** [comment]
Still in draft

File: nas_spec/components/schemas/Payments/ProcessingSettings.yaml [hunk 1]
```diff
@@ -27,39 +43,6 @@ properties:
       - 'No_show'
       - 'Reauthorization'
     description: Indicates the reason for a merchant initiated payment request.
-  # dlocal:
```
Comment (yury-patlyakevich-cko) [inline nas_spec/components/schemas/Payments/ProcessingSettings.yaml:30]: Do we need to remove this properties?

File: nas_spec/components/schemas/Payments/ProcessingSettings.yaml [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
 properties:
-  # mid:
-  #   type: string
-  #   description: Overrides the default merchant/acceptor identifier (MID) configured on your account
-  #   example: "1234567"
-  # senderInformation:
-  #   type: object
-  #   description: |
-  #     Allows you to send additional data required for Visa Direct Original Credit Transaction (OCT) and Mastercard Moneysend payments.
+  order_id:
```
Comment (yury-patlyakevich-cko) [inline nas_spec/components/schemas/Payments/ProcessingSettings.yaml:4]: We need to add max length property for some items from the doc

File: nas_spec/components/schemas/Payments/ProcessingSettings.yaml (hunk 1 above)
Comment (duncan-scuffell-cko) [reply nas_spec/components/schemas/Payments/ProcessingSettings.yaml:30]: The PR was created June 22nd 2021

**Review by yury-patlyakevich-cko — APPROVED** [review]

File: nas_spec/components/schemas/CustomerRequest.yaml [hunk 3]
```diff
@@ -17,3 +17,8 @@ properties:
     description: The customer's name
     maxLength: 255
     example: 'Bruce Wayne'
+  tax_number:
+    type: string
+    description: Contains the customer’s value-added tax registration number.
+    maxLenght: 13
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/CustomerRequest.yaml:23]:
```suggestion
    maxLength: 13
```

File: nas_spec/components/schemas/CustomerRequest.yaml [hunk 4]
```diff
@@ -17,3 +17,8 @@ properties:
     description: The customer's name
     maxLength: 255
     example: 'Bruce Wayne'
+  tax_number:
+    type: string
+    description: Contains the customer’s value-added tax registration number.
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/CustomerRequest.yaml:22]:
```suggestion
    description: Contains the customer’s value-added tax (VAT) registration number.
```

File: nas_spec/components/schemas/Payments/CaptureRequest.yaml [hunk 5]
```diff
@@ -19,6 +19,28 @@ properties:
...
+        description: The shipping address
+        allOf:
+          - $ref: '#/components/schemas/Address'
+      phone:
+        description: The phone number associated with the shipping address
+        allOf:
+          - $ref: '#/components/schemas/PhoneNumber'
+      from_address_zip:
+        type: string
+        description: Postal Code/Zip the location being shipped from.
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/CaptureRequest.yaml:38]:
```suggestion
        description: The postal code or zip code of the location being shipped from.
```

File: nas_spec/components/schemas/Payments/Items.yaml [hunk 6]
```diff
@@ -24,6 +24,28 @@ items:
...
+      description: Unit of Measure code used for an item in transaction.
+      maxLength: 12
+      example: 'metres'
+    total_amount:
+      type: number
+      description: Total Amount for the line item
+      example: 29000
+    tax_amount:
+      type: number
+      description: Contains the total amount of sales tax or value added tax (VAT) on the total purchase amount. Tax should be included in the total purchase
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/Items.yaml:43]:
```suggestion
      description: Contains the total amount of sales tax or value-added tax (VAT) on the total purchase amount. Tax should be included in the total purchase.
```

File: nas_spec/components/schemas/Payments/PaymentRequest.yaml [hunk 7]
```diff
@@ -77,6 +77,10 @@ properties:
         description: The phone number associated with the shipping address
         allOf:
           - $ref: '#/components/schemas/PhoneNumber'
+      from_address_zip:
+        type: string
+        description: Postal Code/Zip the location being shipped from.
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/PaymentRequest.yaml:82]:
```suggestion
        description: The postal code or zip code of the location being shipped from.
```

File: nas_spec/components/schemas/Payments/CaptureRequest.yaml [hunk 8]
```diff
@@ -19,6 +19,28 @@ properties:
...
     maxLength: 50
     example: 'ORD-5023-4E89'
+  customer:
+    $ref: '#/components/schemas/CustomerRequest'
+  shipping:
+    type: object
+    description: The shipping details
+    properties:
+      address:
+        description: The shipping address
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/CaptureRequest.yaml:29]:
```suggestion
        description: The shipping address.
```

File: nas_spec/components/schemas/Payments/CaptureRequest.yaml [hunk 9]
```diff
@@ -19,6 +19,28 @@ properties:
...
+        allOf:
+          - $ref: '#/components/schemas/Address'
+      phone:
+        description: The phone number associated with the shipping address
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/CaptureRequest.yaml:33]:
```suggestion
        description: The phone number associated with the shipping address.
```

File: nas_spec/components/schemas/Payments/Items.yaml [hunk 10]
```diff
@@ -24,6 +24,28 @@ items:
...
+      example: 1000
+    discount_amount:
+      type: number
+      description: Discount applied to each invoice line item
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/Items.yaml:47]:
```suggestion
      description: The discount applied to each invoice line item.
```

File: nas_spec/components/schemas/Payments/ProcessingSettings.yaml [hunk 11]
```diff
@@ -9,6 +9,36 @@ properties:
...
+    maxLength: 17
+    example: '123456789'
+  tax_amount:
+    type: number
+    description: Contains the customer’s value-added tax registration number.
+    minimum: 0
+    example: 3000
+  discount_amount:
+    type: number
+    description: Discount amount applied by the merchant to the transaction.
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/ProcessingSettings.yaml:24]:
```suggestion
    description: The discount amount applied to the transaction by the merchant.
```

**Review by Enrica-Mameli-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — DISMISSED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Review by yury-patlyakevich-cko — APPROVED** [review]

**Review by Enrica-Mameli-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Pull request merged**